app.js

```diff
@@ -1,81 +1,48 @@
 'use strict'
 
 const io = require('socket.io-client')
-const readline = require('readline')
-const rooms = require('./rooms.js')
+const Rooms  = require('./rooms.js')
+const Input = require('./input.js')
+const Colors = require('./color.js')
+
 const local = 'http://localhost:3030'
 const heroku = 'https://hacker-chatroom.herokuapp.com'
-const colors = require('./color.js')
 
-let username = 'anonymous'
 // For local testing: http:localhost:3030
-const socket = io.connect(heroku)
+const socket = io.connect(local)
 // const socket = io.connect('http://localhost:3030')
 
-let rl = readline.createInterface({
-    input: process.stdin,
-    output: process.stdout
-});
+// TODO: Enter username
 
 socket.on('connect', (data) => {
-    socket.emit('create', 'room1')
     console.log('Connected to Server')
-    rooms.getRooms.then((data) => {
-        console.log(Object.keys(data));
+    Rooms.getRooms(socket).then((chatRooms) => {
+        console.log('Available rooms:')
+        console.log(chatRooms)
+        if (chatRooms.length === 1) {
+            console.log(`Joined ${chatRooms[0]}`)
+            socket.emit('create', chatRooms[0])
+        }
+    // TODO: Select room or create room
+    Input.sendMessage(socket)
     }).catch((err) => {
-        console.log(err);
+        console.error('Error getting rooms', err)
```

Comment:
Opps it was a mistake but the benefit is that it would print to `stderr` instead of `stdout` but we don't use `stderr` so I'll remove it